js/app/controllers/importcontroller.js

```diff
@@ -122,7 +124,23 @@ app.controller('ImportController', ['$scope', '$filter', 'CalendarService', 'VEv
 				$scope.$apply();
 				$scope.closeIfNecessary();
 
-				//TODO - refetch calendar
+				CalendarService.getAll().then(function(calendars) {
```

Comment:
This should really not be necessary. You are reloading all calendars once for each ics file the user selects.
Let me look into a better solution.